src/worker/indexedDBAccess.ts

```diff
@@ -0,0 +1,43 @@
+import Dexie from "dexie";
+
+const DB_NAME = "dictionary_sources";
+const VERSION = 1;
+
+/**
+ * files Table Scehma
+ * +-------+
+ * | id    |
+ * +-------+
+ * | name  |
+ * +-------+
+ * | file  |
+ * +-------+
```

Comment:
Good diagram! Could you mark which one is the primary key (e.g., indexed)? I believe `id` is an autoincrement primary key, right?

Reply:
Yes